### ISSUE TITLE
strange behavior when seeding directories

### ISSUE THREAD

**Issue by devTristan** [feross/webtorrent#349]
#### Seeding a directory with one file in it:

Same script as above

##### Expected:
```
$ node seed.js 
[ 'a.txt' ]
```

##### Actual:
```
$ node seed.js 
[ 'www' ]
```

#### Seeding a directory with two files in it:

```javascript
var WebTorrent = require('webtorrent-hybrid');

var client = new WebTorrent();

client.seed('./www', function(torrent){
  console.log(torrent.files.map(file => file.path));
});
```

##### Expected:
```
$ node seed.js 
[ 'a.txt', 'b.txt' ]
```

##### Actual:
```
$ node seed.js 
[ 'www/a.txt', 'www/b.txt' ]
```

**Comment by feross** [comment]
Thanks for reporting. There was indeed a bug when seeding a directory with one file in it. That's fixed in create-torrent 3.13.0 now. :+1: That was actually quite tricky to debug! https://github.com/feross/create-torrent/commit/b4592987a2fa128d3848262e03c0ce80c42b5cd3

The path property on the file automatically includes the directory name that should be created when saving the file to the filesystem. So, your second example of seeding a directory with two files is actually behaving as expected. Maybe we shouldn't do that, but it's actually expected behavior. If you want to discuss that further, open a new issue. :)

**Comment by devTristan** [comment]
Awesome, thanks :)
